src/cli/eofwrap.py

```diff
@@ -232,7 +232,14 @@ def _short_exception_msg(self, e: Exception):
         return short
 
     def _wrap_fixture(self, fixture: BlockchainFixture, traces: bool):
-        env = Environment()
+        env = Environment(
```

Comment:
This set of arguments taken from BlockchainTest.make_genesis - take whatever `env` is used for there